include/flag_gems/operators.h

```diff
@@ -14,7 +14,11 @@ at::Tensor sum_dim(const at::Tensor &self,
                    at::OptionalIntArrayRef dim,
                    bool keepdim = false,
                    ::std::optional<at::ScalarType> dtype = ::std::nullopt);
-
+std::tuple<at::Tensor, at::Tensor> max_dim(const at::Tensor &self,
+                                           at::OptionalIntArrayRef dim,
+                                           bool keepdim,
+                                           ::std::optional<at::ScalarType> dtype);
+at::Tensor max(const at::Tensor &self);
```

Comment:
We also need to register those operators to torch libraries `flag_gems` and `aten`.

See an example in `https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/csrc/cstub.cpp` for registering to `flag_gems`

```c++
TORCH_LIBRARY(flag_gems, m) {
  // blas
  m.def("addmm(Tensor self, Tensor mat1, Tensor mat2, *, Scalar beta=1, Scalar alpha=1) -> Tensor");
}
```

and
```c++
TORCH_LIBRARY_IMPL(flag_gems, CUDA, m) {
  // blas
  m.impl("addmm", TORCH_FN(addmm));
}
```

and also registering to `aten`

https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/csrc/aten_patch.cpp

```c++
namespace flag_gems {
TORCH_LIBRARY_IMPL(aten, CUDA, m) {
  REGISTER_AND_LOG("addmm", addmm);
  REGISTER_AND_LOG("bmm", bmm);
  REGISTER_AND_LOG("mm", mm_tensor);
}
```